data/Language/JSON.yaml

```diff
@@ -0,0 +1,14 @@
+identifier: JSON
+wikidata: Q2063
+full_name: JavaScript Object Notation
+extensions: 
+  - json
+aliases:
+  - json
+line_continuation: ''
```

Comment:
What does this mean?

Does JSON have a line continuation token?
If not, remove this property.

Reply:
Alright, so we just drop the properties if it's not used?